examples/servlet/src/main/java/dev/openfga/example/config/SecurityConfig.java

```diff
@@ -0,0 +1,74 @@
+package dev.openfga.example.config;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configurers.CsrfConfigurer;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.AuthenticationConverter;
+import org.springframework.security.web.authentication.AuthenticationFilter;
+import org.springframework.security.web.context.SecurityContextHolderFilter;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+@Configuration
+@EnableMethodSecurity
+public class SecurityConfig {
+
+    private static final String TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJodHRwczovL3Rlc3QuYXV0aDAuY29tLyIsInN1YiI6ImFubmUiLCJhdWQiOiJodHRwczovL3F1aWNrc3RhcnRzL2FwaSIsImlhdCI6MTcxNDY1NzE5OCwiZXhwIjoxNzE0NzQzNTk4fQ.wbpLJa5Q1196V2Of--XuvGc-ONHNZOZTSrwWLoD3isQ";
```

Comment:
Minor - can you add a comment about why we're doing this?

(e.g. this is not for prod, it's to make trying out the example easier etc..)